masonry/examples/tests.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2025 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+//! This module imports widget code from other examples.
+//!
+//! When running unit tests, it will compare the rendered snapshots of the widgets
+//! with the expected snapshots.
+
+fn main() {
+    println!("This example is only used to compile other examples.");
+}
+
+#[cfg(test)]
+#[allow(dead_code, reason = "We don't need to run the main functions.")]
+#[allow(missing_docs, reason = "Example code doesn't need docs.")]
+#[path = "."]
+pub mod others {
```

Comment:
If you run `cargo nextest run --examples`, do the versions of the tests in this example race with the versions in the main examples.

That is, is `assert_render_snapshot` written to be process safe, in both the success and failure cases?

Reply:
Yeah, I don't think it's too important. This is not a blocking concern, just something that comes up

Reply:
I've changed the code structure, this should no longer be a problem.